test/include/vectorpostprocessors/ChannelGradientVectorPostprocessor.h

```diff
@@ -0,0 +1,61 @@
+#ifndef CHANNELGRADIENTVECTORPOSTPROCESSOR_H
+#define CHANNELGRADIENTVECTORPOSTPROCESSOR_H
+
+#include "GeneralVectorPostprocessor.h"
+
+// Forward Declarations
+class ChannelGradientVectorPostprocessor;
+
+template <>
+InputParameters validParams<ChannelGradientVectorPostprocessor>();
+
+/**
+ *  ChannelGradientVectorPostprocessor is a VectorPostprocessor that calculates
+ *  the difference between two LineValueSampler vector postprocessors.
+ */
+
+class ChannelGradientVectorPostprocessor : public GeneralVectorPostprocessor
+{
+public:
+  /**
+    * Class constructor
+    * @param parameters The input parameters
+    */
```

Comment:
Please delete boilerplate comments if they aren't specific from the class. Usually you don't need to document override methods at all. However, new methods should be documented.